docs/getting-started/box-basics.md

```diff
@@ -26,90 +26,90 @@ Since `box` is considered a complete box management script, its functions requir
+When run alone, `box` will start a guided tutorial with install and remove options for availible applications. This option is intended for unfamiliar Swizzin or Linux users. Most users will find `install`, `remove` and `upgrade` to be more viable options, after the initial setup procress.
 
 Syntax:
 ```bash
 box
 ```
 
 ## help
-The function of help is basic: provide you usage details on how to use box and the commands available to you.
+This function provides basic information and usage examples for all availible box commands.
 
 Syntax:
 ```bash
 box help
 ```
 
 ## install
-The function used to install applications from the server. If you need help with specific commands, they are available on the application sub-pages here. You can specify as many applications is you like here.
+This function installs applications on the server. If you need help with specific commands, please see the `Applications` section on the left side of the website. You can specify multiple applications to install at the same time.
 
 Syntax:
 ```bash
 box install rtorrent deluge
 ```
 
 ## remove
-The function used to remove applications from the server. If you need help with specific commands, they are available on the application sub-pages here. You can specify as many applications is you like here.
+This function removes applications from the server. If you need help with specific commands, please see the `Applications` section on the left side of the website. You can specify multiple applications to remove at once.
 
 Syntax:
 ```bash
 box remove rtorrent deluge
 ```
 
 ## update
-This function can be used to update your swizzin repository. There are a number of update scripts that trigger each time this command is called -- these will typically help fix issues or upgrade your current installation to a new method of handling the application in question.
+This function updates to the latest swizzin release. This typically fixes issues with the install procress. It is often a requirement for upgrading applications or fixing broken aspects of the seedbox solution.
 
 Syntax:
 ```bash
 box update
 ```
 
 ## upgrade
-This function can be used to upgrade a specific application to a newer version or recompile an application which was compiled during install. Only selected applications have upgrade scripts. You can check which scripts have `upgrade` components by looking in the `/etc/swizzin/scripts/upgrade` directory. Call them directly by name.
+This function upgrades a specific application to a newer version. It also recompiles applications without versions to install the latest fixes. Only some applications have upgrade scripts and their name is required to upgrade them. You can view the specialized list on GitHub. https://github.com/swizzin/swizzin/tree/master/scripts/upgrade
```

Comment:
```suggestion
This function upgrades a specific application to a newer version. It can also recompile applications without versions to install the latest fixes. Only some applications have upgrade scripts and their name is required to upgrade them. You can view the specialized list on GitHub. https://github.com/swizzin/swizzin/tree/master/scripts/upgrade
```